md/widgets.md

```diff
@@ -162,7 +177,10 @@ You can use a variable to hold the resulting data after success or failure.
 
 For pages that are displayed in a task or process context, the button widget can be used to submit a form, completing the human task or starting a process instance. For form submission, you need to define only the data to send. The URL is extracted from the context.
 
-Finally, you can use the widget button to add or remove a data from a given collection.
+Another possible use of the widget button is to add or remove a data from a given collection.
+
+Finally, you can use a button to open a modal container (pop up window) on the page, as long as the modal ID is specified into the appropriate field. Another action for the button widget is to close the modal that contains it.
+Other actions can also close the modal if they succeed, being enabled when the `Close modal on success` property is positive.
```

Comment:
What is implied in "Other actions"? Can you give an example?
Nice to have: Try being even clearer so as what Fred needs to do. For example (but not sure this is the exact way it is implemented in the UI Designer): "... can close the modal on success. To do so, set the `Close modal on success` property to `yes`.